packages/sdk/src/chains/Substrate/utils/depositFns.ts

```diff
@@ -162,9 +162,12 @@ export const handleTxExtrinsicResult = (
 export const createDestIdMultilocationData = (address: string, domainId: string): object => ({
   parents: 0,
   interior: {
-    x2: [
-      { generalKey: [20, address.padEnd(66, '0')] },
-      { generalKey: [1, numberToHex(Number(domainId)).padEnd(66, '0')] },
+    x3: [
```

Comment:
I feel like this will break local and fix it on testnet.
We need something more future proof.

Reply:
I think for now we should at least have one more argument passed to `createDestIdMultilocationData` function that defines environment. This way we can have different logic based on environment.